mono/mini/Makefile.am.in

```diff
@@ -1,6 +1,8 @@
 count=100000
 mtest=for_loop
 monodir=$(top_builddir)
+mono=$(if $(MONO_EXECUTABLE),$(MONO_EXECUTABLE),mono)
+sep=$(if $(LIST_SEPARATOR),$(LIST_SEPARATOR),:)
```

Comment:
Here is a problem: If we are running this from Cygwin, we need to potentially be able to support EITHER running a cygwin-built mono OR a msvc-built mono. Will the cygwin-built mono use ; or : as its separator?